src/MeadeCommandProcessor.cpp

```diff
@@ -496,6 +496,25 @@ bool gpsAqcuisitionComplete(int &indicator);  // defined in c72_menuHA_GPS.hpp
 //      Returns:
 //        "1" if successfully scheduled
 //
+// :MHR#
+//      Description:
+//        Home RA stepper via Hall sensor
+//      Information:
+//        This attempts to find the hall sensor and to home the RA ring accordingly.
+//      Parameters:
+//        If the last letter 'R' is uppercase the mount will search east first, if it is lowercase it will search west first.
```

Comment:
mh ... east and west are hemisphere dependant if i understand it correctly, clockwise/counterclockwise or left/right would be more intuitive. Especially since auto-homing does not depend on the sky orientation at all ... it is a pure mount feature.

Reply:
Good point.